.circleci/config.yml

```diff
@@ -154,15 +139,16 @@ jobs:
       executor: << parameters.os >>
       environment:
          PKG_NAME: "cmor"
-         VERSION: "3.6.1"
+         VERSION: "3.7.0"
          PYTHON_VERSION: << parameters.python_version >>
-         CHANNELS: "-c cdat/label/nightly -c conda-forge -c cdat"
+         CHANNELS: "-c conda-forge -c cdat/label/nightly -c cdat"
```

Comment:
@mauzey1 do we still have a `cdms2` dependency? Is it just for the test suite?

Reply:
@durack1 cdms2 is not a dependency, it is just supported by CMOR.  The test suite was having issues because it was installing an older version of cdms2; the conda-forge version is newer than the nightly CDAT version.

Reply:
@mauzey1 no problem, yes I think let's depend on the `conda-forge` channel version, it will likely be the only one supported until end of life